apis/apps/v1beta3/emqxenterprise_types.go

```diff
@@ -22,8 +22,12 @@ import (
 )
 
 type License struct {
-	Data       []byte `json:"data,omitempty"`
+	// Data represents the byte data of the license content
+	Data []byte `json:"data,omitempty"`
+	// StringData represents the string data of the license content
```

Comment:
stringData allows specifying non-binary secret data in string form. It is provided as a write-only input field for convenience. All keys and values are merged into the data field on write, overwriting any existing values.